package/skelcd-control-SLED.changes

```diff
@@ -1,8 +1,13 @@
+-------------------------------------------------------------------
+Tue Dec 11 16:26:53 UTC 2018 - dgonzalez@suse.com
+
+- Use FHS compliant path (fate#325482)
+- 15.1.0
+
 -------------------------------------------------------------------
 Thu Nov 29 13:47:51 UTC 2018 - Stasiek Michalski <hellcp@mailbox.org>
 
 - Update yast2-theme dependency (boo#1108422)
-- 15.0.23
```

Comment:
Just a question: Why changing the history? Any reason for that?

Reply:
Yes. That @hellcp's change was submitted to master but not to SLE-15-GA. So it was an error back then to use the version number 15.0.23. That version implies the same change should be in both branches... which is not true.

Reply:
This change was introduced at the time to fix the version number. See https://github.com/yast/skelcd-control-SLED/pull/89/commits/c2b7afe9f8ace1f78a0b81c84dbb68fb15a38923

>  Fix version number
>
> Because the package has diverged (since commit d12ab2f), the minor
> version is increased. Also fix the log entries to keep consistence.
> 
> More info:
> 
> - https://yastgithubio.readthedocs.io/en/latest/versioning/

Reply:
OK, thanks for the info!